[Marketplace Contribution] - EXPANDR-7038 - Azure Resource Graph (#35326)
demisto/content: Packs/AzureResourceGraph/Integrations/AzureResourceGraph/AzureResourceGraph_description.md

Azure Resource Graph is an Azure service designed to extend Azure Resource Management by providing efficient and performant resource exploration with the ability to query at scale across a given set of resources.

Full documentation for this integration is available in the [reference docs](https://xsoar.pan.dev/docs/reference/integrations/azure-resource-graph).

> 
> 

Reviewer: ```suggestion
```

> There are two application authentication methods available.
> Follow your prefered method's guide on how to use the admin consnt flow in order to recieve your authentication information:
> 
> * [Cortex XSOAR Application](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#cortex-xsoar-application)
> * [Self-Deployed Application](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#self-deployed-application)

Reviewer: ```suggestion

Follow the [Self-Deployed Application Authentication guide](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#self-deployed-application)  to configure and enable authentication.
```
